docs/manual/docs/cmd/graph/teams/teams-team-add.md

```diff
@@ -0,0 +1,42 @@
+# graph teams add
```

Comment:
This file should be referenced from mkdocs.yml or it won't be included in the generated documentation.